sunbird-rc-esignet-integration-impl/README.md

```diff
@@ -15,32 +15,27 @@ mosip.esignet.integration.key-binder=MockKeyBindingWrapperService
 mosip.esignet.integration.audit-plugin=LoggerAuditService
 mosip.esignet.integration.captcha-validator=GoogleRecaptchaValidatorService
 
+## -------------------------------------------- IDP-UI config -----------------------------------------------------------------##
+mosip.esignet.authenticator.default.auth-factor.kba.field-details={{'id':'policyNumber', 'type':'text', 'format':''},{'id':'name', 'type':'text', 'format':''},{'id':'dob', 'type':'date', 'format':'dd/mm/yyyy'}}
+mosip.esignet.authenticator.default.auth-factor.kba.individual-id-field=policyNumber
```

Comment:
We donot have to separate above 2 properties.

Reply:
done